rotkehlchen/types.py

```diff
@@ -837,6 +838,57 @@ def __str__(self) -> str:
         return f'Addressbook entry with name "{self.name}", address "{self.address}" and blockchain {str(self.blockchain) if self.blockchain is not None else None}'  # noqa: E501
 
 
+class LocationAssetMappingDeleteEntry(NamedTuple):
+    location: 'Location | None'
+    location_symbol: str
+
+    def serialize(self) -> dict[str, str | None]:
+        return {
+            'locatoin': None if self.location is None else self.location.serialize(),
+            'location_symbol': self.location_symbol,
+        }
+
+    @classmethod
+    def deserialize(cls: type['LocationAssetMappingDeleteEntry'], data: dict[str, Any]) -> 'LocationAssetMappingDeleteEntry':  # noqa: E501
+        """May raise:
+        -KeyError if required keys are missing
+        """
+        return cls(
+            location=data['location'],
+            location_symbol=data['location_symbol'],
+        )
+
+    def __str__(self) -> str:
+        return f'Asset mapping entry with asset location {self.location} and location_symbol "{self.location_symbol}"'  # noqa: E501
```

Comment:
1. Better wording.
2. Why do you need this? Where is it used?

```suggestion
        return f'Location asset mapping for {self.location} and location_symbol "{self.location_symbol}"'  # noqa: E501
```